src/services/location-service.ts

```diff
@@ -0,0 +1,45 @@
+import { distanceBetweenTwoPoints } from '@/services/util/DistanceCalculator';
+
+const SKOOKUM_COORDINATES = {
+  latitude: 39.758117, // default to skookum coordinates
+  longitude: -104.989888, // default to skookum coordinates
+};
+
+export class LocationService {
+
+  /**
+   * Request permission to get the user's location while the
+   * browser page is active.
+   */
+  public requestLocation(): Promise<any> {
+    return new Promise<any>((res, rej) => {
+      res(true);
+    });
+  }
+
+  /**
+   * Get the user's current location, if the user
+   * has accepted location permissions.
+   */
+  public get(): Promise<any> {
+    return new Promise<any>((res, rej) => {
+      res(SKOOKUM_COORDINATES);
+    });
+  }
+
+  public async isWithinGeoRadius(locationRadius: number, latitude: number, longitude: number): Promise<boolean> {
```

Comment:
This should handle the geoRadius calculations for us.  Little harder than anticipated, but luckily google easily found me a `distanceCalculator` that we could utilize.